cmd/internal/cli/build.go

```diff
@@ -224,3 +225,53 @@ func makeDockerCredentials(cmd *cobra.Command) (authConf *ocitypes.DockerAuthCon
 
 	return
 }
+
+// remote builds need to fail if we cannot resolve remote URLS
+func handleRemoteBuildFlags(cmd *cobra.Command) {
+	// if we can load config and if default endpoint is set, use that
+	// otherwise fall back on regular authtoken and URI behavior
+	e, err := sylabsRemote(remoteConfig)
+	if err == nil {
```

Comment:
Can we do this instead:

```
if err == scs.ErrNoDefault {
    sylog.Warningf(...)
    return
} else if err != nil {
    sylog.Fatalf(...)
}

[...]
```

This way we avoid having things living too deep inside of conditional blocks